collectors/charts.d.plugin/speedtest/README.md

```diff
@@ -0,0 +1,76 @@
+<!--
+title: "Speedtest monitoring with Netdata"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/charts.d.plugin/speedtest/README.md
+sidebar_label: "Speedtest"
+-->
+
+# Speedtest monitoring with Netdata
+
+[Speedtest CLI](https://www.speedtest.net/apps/cli) is a Linux native command line tool used to monitor internet connection performance. 
+
+This module monitors results and metrics related to [Speedtest](https://www.speedtest.net/apps/cli).
+
+## Requirements
+
+For all nodes you are going to monitor speedtest results from:
+- Install Speedtest CLI by following the installation instructions mentioned [here](https://www.speedtest.net/apps/cli).
+- Accept the license agreement as the Netdata by running `speedtest` from the command line
+
+```bash
+sudo -u netdata speedtest
+```
+
+## Metrics
+
+All metrics have "speedtest." prefix.
+
+| Metric              |    Scope    |  Dimensions   |     Units     |
+|---------------------|:-----------:|:-------------:|:-------------:|
+| download            |   global    |  bandwidth    |  bps          |
+| upload              |   global    |  bandwidth    |  bps          |
+| packetloss          |   global    |  packet loss  |  packets      |
+| idle_latency        |   global    |  latency      |  milliseconds |
+| download_latency    |   global    |  latency      |  milliseconds |
+| upload_latency      |   global    |  latency      |  milliseconds |
+| idle_jitter         |   global    |  jitter       |  milliseconds |
+| download_jitter     |   global    |  jitter       |  milliseconds |
+| upload_jitter       |   global    |  jitter       |  milliseconds |
+| download_bytes      |   global    |  bandwidth    |  bytes        |
+| upload_bytes        |   global    |  bandwidth    |  bytes        |
+
+## Configuration
+
+Edit the `charts.d/speedtest.conf` configuration file using `edit-config` from the Netdata [config
```

Comment:
Are you going to add  `speedtest.conf` at `netdata/netdata`? It will be good to have an example file there.

Reply:
I will do it after this PR is merged.

Reply:
Thanks!